doc/sphinx-guides/source/_static/api/add-storage-site.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
It's awesome that the PDF actually builds but we don't want it checked into the source code.

(This comment is for `doc/sphinx-guides/source/_static/Dataverse.pdf` but I can't leave a comment there.)